src/backend/distributed/utils/citus_ruleutils.c

```diff
@@ -1004,6 +1022,42 @@ AppendOptionListToString(StringInfo stringBuffer, List *optionList)
 }
 
 
+/*
+ * AppendStorageParametersToString converts the storage parameter list to its
+ * textual format, and appends this text to the given string buffer.
+ */
+static void
+AppendStorageParametersToString(StringInfo stringBuffer, List *optionList)
+{
+	if (optionList != NIL)
```

Comment:
We have the same `if` statement at the call site. What's the appropriate place for it?